src/lib/eina/eina_log.c

```diff
@@ -36,7 +32,6 @@
 #endif
 
 #ifdef _WIN32
-# include <windows.h>
```

Comment:
Why removing the explicit `windows.h` include?

Reply:
> `evil_private.h` includes `<evil_windows.h>`, which uses `WIN32_LEAN_AND_MEAN`.